src/main/drivers/light_ws2811strip.c

```diff
@@ -42,6 +42,8 @@
 #include "drivers/dma.h"
 #include "drivers/io.h"
 
+#include "io/ledstrip.h"
```

Comment:
The driver layer should not have dependencies into the 'upper' layers.

Reply:
Fixed